PM2021-004 - ADFS outage.md

```diff
@@ -0,0 +1,50 @@
+# Incident Title
+
+ADFS Outage
```

Comment:
I had a look at some of the earlier postmortems, at it looks like the "#Incident title" should be replaced with the actual title. 

Before:
```
# Incident Title

ADFS Outage

## Summary
```

After:
```
# ADFS Outage

## Summary
```

Reply:
Good spot!

Reply:
Amended